frontend/src/translations/ru.global.json

```
@@ -0,0 +1,550 @@
+        "tokens_burned": "Использовано токенов"
+      }
+    },
+    "actions": {
+      "actions": "Действия"
+    },
+    "receipts": {
+      "receipt": "Квитанция",
+      "receipts": "Квитанции",
+      "receipt_id": "ID квитанции",
+      "executed_in_block": "Выполнено в блоке",
+      "predecessor_id": "ID предшественника",
+      "receiver_id": "ID получателя",
+      "status": {
+        "Unknown": "Неизвестен",
+        "Failure": "Сбой",
+        "SuccessValue": "Одобрено",
+        "SuccessReceiptId": "Одобрено"
+      }
+    },
+    "contracts": {
+      "contract": "Контракт"
+    },
+    "nodes": {
+      "title": "Узлы"
+    },
+    "stats": {
+      "title": "Статистика",
+      "title_charts_and_stats": "Графики/Статистика",
+      "daily": "По дням",
+      "weekly": "По неделям",
+      "total": "Общих",
+      "unique": "Уникальных",
+      "live": "Текущая"
+    },
+    "unit": {
+      "seconds": "сек"
+    },
+    "state": {
+      "not_available": "Нет информации",
+      "yes": "Да",
+      "no": "Нет"
+    },
+    "date_time": {
+      "date_time_format": "DD MMMM YYYY [в] h:mm:ss",
+      "date_format": "DD MMMM YYYY",
+      "time_format": "[в] h:mm:ss"
+    }
+  },
+  "page": {
+    "home": {
+      "title": {
+        "explore": "Исследуйте",
+        "near_blockchain": "<br />NEAR Блокчейн."
+      }
+    },
+    "accounts": {
+      "account": "Аккаунт",
+      "error": {
+        "account_fetching": "На данный момент информация недоступна. Пожалуйста, проверьте правильность имени учетной записи или попробуйте позже.",
+        "account_not_found": "Аккаунт ${account_id} не найден. Пожалуйста, проверьте правильность имени учетной записи или попробуйте позже."
+      }
+    },
+    "blocks": {
+      "title": "Блоки",
+      "error": {
+        "block_fetching": "На данный момент информация недоступна. Пожалуйста, проверьте правильность хеша блока или попробуйте позже."
+      }
+    },
+    "transactions": {
+      "transaction_execution_plan": "План выполнения транзакции",
+      "error": {
+        "transaction_fetching": "На данный момент информация недоступна. Пожалуйста, проверьте правильность хеша транзакции или попробуйте позже."
+      }
+    }
+  },
+  "component": {
+    "nodes": {
+      "Validators": {
+        "location": "Местоположение",
+        "validator": "Валидатор",
+        "fee": "Комиссиия",
+        "delegators": "Вкладчики",
+        "stake": "Размер стейка",
+        "cumulative_stake": "Кумулятивная доля"
+      },
+      "ValidatorRow": {
+        "warning_tip": "Совокупная доля валидатора 1 - ${node_tip_max} превышает 33%. Передача NEAR для стейкинга указанным ниже валидаторам улучшает децентрализацию сети."
+      },
+      "ValidatorMainRow": {
+        "state": {
+          "active": {
+            "title": "Активный",
+            "text": "текущие проверочные узлы"
+          },
+          "joining": {
+            "title": "Соединение",
```

```suggestion
            "title": "Присоединяющийся",
```
But I'm not sure about it. @frol @telezhnaya could you please take a look?

---

We decided to use "Соединение"